Demos/Mouse/main.cpp

```diff
@@ -9,97 +9,150 @@
 #include <LibGUI/Widget.h>
 #include <LibGUI/Window.h>
 #include <LibGfx/Bitmap.h>
+#include <LibGfx/Path.h>
 
-static unsigned s_mouse_button_state;
+#include <math.h>
```

Comment:
```suggestion
#include <LibM/math.h>
```

Reply:
Why? `math.h` is a completely fine (and in fact portable) way of referring to the header

Reply:
good point!